src/components/pages/MainPage.js

```diff
@@ -16,6 +16,7 @@ import './MainPage.css';
 const urlPropsQueryConfig = {
   tsvUrl: { type: UrlQueryParamTypes.string },
   externalAudioUrl: { type: UrlQueryParamTypes.string, queryParam: 'audioUrl' },
+  startTimestamp: { type: UrlQueryParamTypes.string, queryParam: 't' }
```

Comment:
Please use number as the type, not string. Then you do not need this parseFloat nonsense below.

Reply:
Done